pygsti/layouts/prefixtable.py

```diff
@@ -1208,6 +1207,8 @@ def _draw_graph(G, node_label_key='label', edge_label_key='promotion_cost', figu
         An optional size specifier passed into the matplotlib figure
         constructor to set the plot size.
     """
+    import matplotlib.pyplot as plt
```

Comment:
Moved import that broke minimal install. Matplotlib is an optional dependency currently.